src/pkg/logger/logger.go

```diff
@@ -99,6 +115,18 @@ var (
 	DestinationNone Destination = io.Discard
 )
 
+// can't define method on Destination type
+func destinationString(d Destination) string {
```

Comment:
Similar problem to `Level` above, there's nothing wrong with this approach but it may run into issues if we need to grow it. We have some application logic not strictly covered by `io.Writer` so we may have to add to our `Desination` type at some point. Nonblocking, just something to consider.